auto/cgroup

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) Andrew Clayton
+# Copyright (C) F5, Inc.
+
+NXT_HAVE_CGROUP=NO
+
+if [ -f "/proc/mounts" ]; then
+    CGROUP=$(grep cgroup2 /proc/mounts | cut -d " " -f 2)
```

Comment:
```suggestion
    CGROUP=$(grep cgroup2 /proc/mounts | head -n1 | cut -d " " -f 2)
```
even if not so common, there could be multiple cgroup2 mount points in the system, and if that's the case the `NXT_CGROUP_ROOT` will contain an invalid path.

```
# grep cgroup2 /proc/mounts
cgroup2 /sys/fs/cgroup cgroup2 rw,nosuid,nodev,noexec,relatime 0 0
cgroup2 /tmp/cgroup2 cgroup2 rw,relatime 0 0
# CGROUP=$(grep cgroup2 /proc/mounts | cut -d " " -f 2)
# echo $CGROUP
/sys/fs/cgroup /tmp/cgroup2
```
luckily all mount points are valid, the cgroup hierarchy is system-wide after all, but as you can have it mounted in several places, the first entry is commonly the one set up by systemd/init-system.